elastic/kibana
# [index patterns] index pattern create modal

## Summary
Index pattern creation modal - create index patterns from any kibana app. Initial implementation in index pattern management.

Closes https://github.com/elastic/kibana/issues/84725

<img width="1149" alt="Screen Shot 2021-06-28 at 10 07 06 PM" src="https://user-images.githubusercontent.com/216176/123731706-50429b80-d85e-11eb-8583-6c8555d6f230.png">

To test
- Go to index pattern management 
  - Without any data
  - With data but without index patterns
- Click 'Create index pattern'
  - Things should be familiar from here, similar to existing index pattern creation UI 

---
Test with index w/o a time field - 

create an index without a time field -
```
PUT my-index-000001/_doc/1?refresh
{
  "my_id": "1",
  "email": "billgates@ms.com"
}
```
---
Test with a rollup index

- Install ecommerce sample data
- Go to Management / Rollup Jobs and create a new rollup job

### Checklist

Delete any items that are not applicable to this PR.
